src/lighteval/metrics/metrics.py

```diff
@@ -326,6 +327,32 @@ class Metrics(Enum):
         corpus_level_fn=matthews_corrcoef,
         higher_is_better=True,
     )
+    maj_at_5 = SampleLevelMetric(
```

Comment:
FYI `maj_at_4` is the most commonly reported metric for the MATH benchmark - should we add it too?

Reply:
Hi! Yep no problem, but does math require a specific normalisation like GSM8K?